### ISSUE TITLE
Popup not keyboard accessible

### ISSUE THREAD

**Issue by ChrisBAshton** [InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework#45]
Hello - apologies if I'm raising this in the wrong place and I've got the wrong people!

I've seen [this kind of popup in a few places](https://www.itworld.com/article/3136583/macbook-accessories/macbook-pro-thunderbolt-3-adapter-guide-how-to-connect-an-iphone-display-hard-drive-and-more.html) recently:

<img width="787" alt="screen shot 2018-06-21 at 20 48 01" src="https://user-images.githubusercontent.com/5111927/41741937-72777ba6-7594-11e8-9bc7-f15040d0bab5.png">

I noticed that the buttons - 'Update Privacy Settings', 'Sounds Good, Thanks' and 'Not Now' _are not keyboard accessible_ (there is no way of tabbing to them). I think we're just missing a `href` attribute:

```html
<a class="fp5lCfuR-1- _2psJxRh1-1- amwo9KVF-1-" automate_uuid="839647d1-16d6-4ccb-ba31-7fba5301a328"><span class="_1MrLfucw-1-">Update Privacy Settings</span></a>
```

Whilst we're here, looks like these buttons have nice `:hover` styles applied to them, can you apply the same thing to `:focus` so that it's clear what's in focus when the buttons are tabbed to via keyboard? 

Finally, worth applying a `tabindex="1"` on the first button to make sure users can navigate to it right away.

If I have raised this in the wrong place, could you please point me in the right direction? I stumbled onto this repo via [this IABEurope blog post](https://www.iabeurope.eu/blog/blog-legitimate-interest-and-consent-can-both-legal-states-be-declared-in-the-framework/), and IABEurope via [this Global Vendors registration page](https://register.consensu.org/), and `consensu.org` via the Network tab of the pages I've been on recently!

Thanks.

**Comment by HeinzBaumann** [comment]
You are at the right place. This is a sample. Please feel free to create a pull request with the required changes. :)

**Comment by ChrisBAshton** [comment]
Hi - I would happily make a PR but I couldn't find the code in this repo 🤔the code looks extremely similar to what's on https://www.itworld.com/ and https://www.pcworld.com/ but their popup seems to be using `<a>` anchor links, and I can only see [references to Button](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/In-App%20Reference/Mobile%20In-App%20Reference%20Implementation/src/components/popup/intro/intro.jsx#L41-L53) in this repo.

Are you saying that sites will implement their own versions of cookie management, taking _inspiration_ from this repo rather than using code from this repo directly? In which case, perhaps I am in the wrong place and should be getting in touch with ITWorld/PCWorld directly...?

**Comment by Brikky** [comment]
@ChrisBAshton There is a reference implementation in this repo, but different companies/sites also develop their own approaches within the spec. You might want to bring it to the attention of those websites, and they can communicate it with whoever is developing/maintaining their CMP.

**Comment by ChrisBAshton** [comment]
Thanks @Brikky, that's actually very useful - it's hard to reverse engineer how this is all set up globally!

I've found a [list of affected websites](https://securitytrails.com/list/ns/ns0.pcworld.com) and they're all owned by IDG - I'm not sure if the implementation is exclusive to IDG-owned sites or if they're using something that is more widespread, but I have now reached out to them via email.

Closing the issue.
